lib/polling.js

```diff
@@ -23,7 +23,13 @@ TradeOfferManager.prototype.doPoll = function(doFullUpdate) {
 		return;
 	}
 
-	if (!this.apiKey || Date.now() - this._lastPoll < 1000) {
+	if(!this.apiKey){
+		this.emit('debug', 'No API Key, shouldn\'t poll' );
+		this._resetPollTimer(1000);
+		return;
+	}
+	else if (this._lastPoll !== 0 && Date.now() - this._lastPoll < 1000) {
+		this.emit('debug', '# Seconds Since Last Poll ' + (Date.now() - this._lastPoll)/1000 );
```

Comment:
Super picky (sorry). This should be regular sentence capitalization, and the number sign is probably not needed:

"Seconds since last poll"